index.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<title>My portfolio</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+<div class="page-wrapper">
+
+<header>
+    <nav class="navbar">
+       <div class="brand">
+            <a href="#" class="logo">My Logo</a>
+       </div>
+       <ul class="nav-list">
+          <li class="list-item">
+            <a href="#">Portfolio</a>
+          </li>
+          <li class="list-item">
+            <a href="#">About</a>
+          </li>
+          <li class="list-item">
+            <a href="#">Contact</a>
+          </li>
+       </ul>
+
+       <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+       </div>
+    </nav>
+</header>
+
+
+   <section class="headline">
+    <h1 class="display-5 fw-bold primary-text">I’m Yannick<br> Glad to see you!</h1>
+    <p>Using a series of utilities, you can create this jumbotron, just like the one in previous versions of Bootstrap. Check out the examples below for 
+    how you can remix and restyle it to your liking.</p>
+
+    <a class="btn-connect"><h4>Let's connect</h4></a>
+    
+        <ul class="media">
+            <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+            <li><a href="#"><i class="fa fa-medium"></i></a></li>
+            <li><a href="#"><i class="fa fa-github"></i></a></li>
+            <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+        </ul>
```

Comment:
Fix the style of the icons 
#### Original design
![right](https://user-images.githubusercontent.com/58376020/125662595-14f726df-970f-4327-bb45-fdf843fdd14e.png)

#### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125662638-be749139-f6ba-48d4-a72e-2aa0b4070512.png)